### ISSUE TITLE
CVSS3.1 isn't fully supported

### ISSUE THREAD

**Issue by unorsk** [haskell/security-advisories#226]
Is it just me or the cvss parser doesn't fully support the cvss31 standard (some metrics [haven't been implemented](https://github.com/haskell/security-advisories/blob/c38596cf4642efed39fa60b6c9cad0a669004d7b/code/cvss/src/Security/CVSS.hs#L270))
These aren't important semantically since their presence doesn't affect the score calculation, but practically the parser would fail parsing a cvss string containing one of these.

**Comment by TristanCacqueray** [comment]
You are right, the parser presently only supports the strings used in the advisories.

**Comment by unorsk** [comment]
> You are right, the parser presently only supports the strings used in the advisories.

Cool, sounds like something I could fix. Probably after https://github.com/haskell/security-advisories/issues/208 gets merged.

**Comment by dancewithheart** [comment]
https://github.com/haskell/security-advisories/pull/322 adds missing `temporalMetrics` and `environmentalMetrics`.

I am not sue if you want to compute temporal score and environmental score.
